src/main/groovy/com/ceilfors/transform/gq/SingletonCodeFlowManager.groovy

```diff
@@ -23,35 +23,50 @@ enum SingletonCodeFlowManager implements CodeFlowListener {
 
     INSTANCE;
 
-    CodeFlowListener codeFlowListener = new GqFile(createGqFile(), true)
+    private CodeFlowPrinter codeFlowPrinter
 
     private File createGqFile() {
         // By default using "/tmp" instead of using java.io.tmpdir for better user usability
         return new File(System.getProperty("GQTMP", "/tmp"), "gq")
     }
 
+    def setGqFile(File file, boolean timestamp) {
+        codeFlowPrinter = new IndentingCodeFlowPrinter(new SyntaxConvertingCodeFlowPrinter(
+                new SimpleSyntaxConverter(),
+                new PrintWriter(new FileCreatingWriter(file)))
```

Comment:
FileWriter won't create the file if the user deleted the file e.g. when the file is getting too long during runtime. Also I was confused on when to close the stream..